test/gui/shared/steps/steps.py

```diff
@@ -588,3 +596,30 @@ def step(context, username):
     # It might take some time for all files to sync and connect to ther server
     snooze(5)
     isUserSignedIn(context, username)
+
+
+@When("the user removes the connection")
```

Comment:
Which connection? If multiple are configured

maybe add `the last created` or `the currently selected` to the sentence.

Reply:
I have refactored this step to `@When('the user removes the connection for user "|any|" and host |any|')` so that we can remove connection for any user we want.